frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -2,13 +2,19 @@
 <div *ngIf="dataLoaded">
   <div *ngIf="performanceSelected; else selectPerformanceView">
     <app-stepper
-        #stepper
-        [ngStyle]="stepper.currentStepIndex == 3 && submitted ? { 'display': 'none' } : {}"
-        [stepperOptions]="stepperOpts"
-        (stepChange)="onStepChange($event)"></app-stepper>
+      #stepper
+      [ngStyle]="
+        stepper.currentStepIndex == 3 && submitted ? { display: 'none' } : {}
+      "
+      [stepperOptions]="stepperOpts"
+      (stepChange)="onStepChange($event)"
+    ></app-stepper>
```

Comment:
I like a lot about your formatting; e.g. that the closing tag is on a separate line makes editing really easy. However, the new style breaks with the standard we have been moving towards. (The original code was all over the place and we haven't been able to fix it everywhere.) How hard would it be to keep the two-tab indent for "line continuation extensions"? (#stepper above used to be indented two tabs but is now only indented one tab).